src/ssl.c

```diff
@@ -6453,6 +6629,47 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
         }
     #endif
 
+    #ifdef WOLFSSL_STATIC_MEMORY
+    #if defined(WOLFSSL_ALLOW_SSLV3) && !defined(NO_OLD_TLS)
+    WOLFSSL_METHOD* wolfSSLv3_client_static(void* heap)
```

Comment:
Additions of functions to set up WOLFSSL_METHOD with using static memory. Am commenting on this because it seems close to code duplication and is worth looking at for potential refactoring.